libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -593,6 +593,65 @@ std::string YulUtilFunctions::resizeDynamicArrayFunction(ArrayType const& _type)
 	});
 }
 
+string YulUtilFunctions::storageArrayPopFunction(ArrayType const& _type)
+{
+	solAssert(_type.location() == DataLocation::Storage, "");
+	solAssert(_type.isDynamicallySized(), "");
+	solUnimplementedAssert(!_type.isByteArray(), "Byte Arrays not yet implemented!");
+	solUnimplementedAssert(_type.baseType()->storageBytes() <= 32, "...");
```

Comment:
```suggestion
	solUnimplementedAssert(_type.baseType()->storageBytes() <= 32, "");
```

Reply:
I've also cleaned up this and the line below.